src/apis/cache.rs

```diff
@@ -1,7 +1,18 @@
-// For now all
+// TODO: Ideally we should remove files carefully, there may be some configuration files as well
+// We should only remove files which are present in sitemap + FPM.ftd + .packages
+
+// Note: If any read request in the system, It should not delete files.
+// In this case need to show the error, can't remove the files.
+
 pub async fn clear(req: actix_web::HttpRequest) -> actix_web::Result<actix_web::HttpResponse> {
-    // Ideally we should remove files carefully, there may be some configuration files as well
-    // We should only remove files which are present in sitemap + FPM.ftd + .packages
+    // Handle options
+    // 1. files=<file-path>
```

Comment:
Don't say `files=<file-path>`, as they you will need separator to separate each path. Use the fact that a parameter can be repeated to achieve the same in more HTTP way:

`/delete?file=a.ftd&file=b.ftd`.